src/main/utilities/generateJButton.java

```diff
@@ -0,0 +1,30 @@
+package main.utilities;
+
+import java.awt.Color;
+
+import javax.swing.BorderFactory;
+import javax.swing.JButton;
+
+public class generateJButton extends JButton {

```

Comment:
This class is a duplicate of GraphicJButton. Add a new constructor to that class instead